lib/elixir/test/elixir/exception_test.exs

```diff
@@ -259,8 +259,8 @@ defmodule ExceptionTest do
       :exit, reason -> reason
     end
 
-    assert Exception.format_exit(reason) ==
-           "exited in: :gen_server.call(:does_not_exist, :hello)\n    ** (EXIT) no process"
+    assert Exception.format_exit(reason) |>
+           String.starts_with?("exited in: :gen_server.call(:does_not_exist, :hello)\n    ** (EXIT) no process:")
```

Comment:
I think we put the `|>` on the end of the line, always on the next line. Here, I'd probably do something like:

```elixir
message = "..."
assert (reason |> Exception.format_exit() |> String.starts_with?(message))
```

or something like that.